bolt/organizations.go

```diff
@@ -13,12 +13,14 @@ import (
 // Ensure OrganizationsStore implements chronograf.OrganizationsStore.
 var _ chronograf.OrganizationsStore = &OrganizationsStore{}
 
-// OrganizationsBucket is the bucket where organizations are stored.
-var OrganizationsBucket = []byte("OrganizationsV1")
+var (
+	// OrganizationsBucket is the bucket where organizations are stored.
+	OrganizationsBucket = []byte("OrganizationsV1")
+	// DefaultOrganizationID is the ID of the default organization.
+	DefaultOrganizationID = []byte("default")
```

Comment:
does this mean that the first non-default organization will have id of 0? if so, that may break things in the client

Reply:
The default org will be under `default` and the first org created will still be `1`

Reply:
ok then that will still break things in the client

Reply:
👍